src/mixins/filing-template-mixin.ts

```diff
@@ -389,12 +389,24 @@ export default class FilingTemplateMixin extends DateMixin {
           identifier: this.getBusinessId,
           legalType: this.getEntityType
         },
-        parties: this.getOrgPeople,
+        parties: null, // applied below
         offices: this.getOfficeAddresses,
         contactPoint: this.getContactPoint
       }
     }
+    // Apply parties to filing
+    {
+      // make a copy so we don't change original array
+      let parties = cloneDeep(this.getOrgPeople)
+
+      // prepare parties
+      parties = isDraft ? parties : this.prepareParties(parties)
```

Comment:
![image](https://github.com/user-attachments/assets/c76edf4e-31d0-4e79-8c74-ccaca19ae22a)
Please see line 1544, where it removes the officer.id

Reply:
OK, so this shows us that `id` is only used for UI purposes (when party is added)?

Reply:
Yes, id is only for UI.

Reply:
OK, I think this comment is resolved.